### ISSUE TITLE
Opening http://localhost:6789/ automatically

### ISSUE THREAD

**Issue by Ayumi08** [mage-ai/mage-ai#2233]
**Is your feature request related to a problem? Please describe.**
It is a little annoying when starting up a new project, `http://localhost:6789/pipelines` does not automatically open in a browser tab.

**Describe the solution you'd like**
Let console open `http://localhost:6789/pipelines` automatically.

**Describe alternatives you've considered**
Only one solution to this??

**Additional context**
N/A

**Comment by bruno-uy** [comment]
I can take this 🙋🏼‍♂️

**Comment by tommydangerous** [comment]
Wow, you’re the best @bruno-uy!!! <3

**Comment by bruno-uy** [comment]
If you're using `pip` or `conda` to run mage it's pretty straightforward. It gets tricky inside a docker container (recommended usage) because it needs to interact with the host's default browser. I'm trying different options without luck. I'll keep you updated.

**Comment by bruno-uy** [comment]
I'm thinking about using the standard `webbrowser` Python library to open the app URL after the server is up (right before [this print](https://github.com/mage-ai/mage-ai/blob/e6d2714430b27ce560bf523dd3a6283b0b60abbc/mage_ai/server/server.py#L271)), but as I mention the `webbrowser` library cannot access the default browser because docker runs in isolation from the host machine.

If you want to access the host file system, you need to mount a volume, expose those folders, and have control of the window system to open the browser (as far as I understood reading some docs).

*TL;DR:* opening the host browser from the docker container is not the best practice, but we can include that when using `pip` or `conda`.

@Ayumi08 @tommydangerous let me know if I'm missing something and what do you think about my proposal.

**Comment by tommydangerous** [comment]
I think that's totally acceptable solution. Thank you so much!!!

**Comment by wangxiaoyou1993** [comment]
Hi @Ayumi08 , this feature is available in version 0.8.29. Let us know whether it works for you.

**Comment by Ayumi08** [comment]
Working very well! Thank you so much!

**Comment by tommydangerous** [comment]
🙌🙌🙌
